### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by erikcorry**

### CODE REVIEW

**Comment by erikcorry** [comment]
I need to specify that this package requires the next version of the SDK, somehow.

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+
+A driver library for WS2812B-compatible pixel strips like Neopixels.
+
+This takes care of the communication with the pixels via a 1-wire
+timing-based protocol.
+
+To use it, you need to feed byte-arrays of red, green and blue pixel
+data to the strips.
+
+# Example
```
Comment (andersjohnsen) [inline README.md]: '##' ?

File: src/pixel_strip.toit [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+  Data is copied out of the byte array, so you can reuse it for the next
+    frame.
+  */
+  abstract output_interleaved -> none
+
+/**
+A driver that sends data to attached WS2812B LED strips, sometimes
+  called Neopixel.  The UART driver is used.
+*/
+class UartPixelStrip extends PixelStrip:
```
Comment (andersjohnsen) [inline src/pixel_strip.toit]: Move to uart.toit

File: src/pixel_strip.toit [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+/**
+A driver that sends data to attached WS2812B LED strips, sometimes
+  called Neopixel.  The I2S driver is used.
+Deprecated.  The current esp-idf versions seem to have a bug
+  where they replay old data instead of sending zeros on the
+  I2S bus.  This makes this driver unreliable, so we are
+  deprecating it until we can resolve the issue.  Please use
+  UartPixelStrip instead.
+*/
+class I2SPixelStrip extends PixelStrip:
```
Comment (andersjohnsen) [inline src/pixel_strip.toit]: Move to i2s.toit

**Review by andersjohnsen — APPROVED** [review]
EPIC!

**Pull request merged**